components/buttons/buttons.scss

```diff
@@ -0,0 +1,301 @@
+.msds-btn {
+  position: relative;
+  display: inline-block;
+  font-weight: $btn-font-weight;
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: bottom;
+  user-select: none;
+  background-color: transparent;
+  line-height: 1.5rem;
```

Comment:
I would prefer pixels over Rem as they are easier to read/understand and adjust according to the design guide. We also have a process thats converting px to rem in the browser already